deploy/kubernetes/examples/demo-storageclass-using-secret-from-config.yaml

```diff
@@ -0,0 +1,21 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: demo-storageclass
+provisioner: block.csi.ibm.com
+volumeBindingMode: WaitForFirstConsumer  # Optional. Required for Topology Awareness.
+parameters:
+  # a pool value MUST be provided for all systems.
+  # non secret related parameters can be provided in by_system_id parameter for each system or by parameter name as default.
+
+  by_system_id: '{"demo-system-id-1":{"pool":"demo-pool-1","SpaceEfficiency":"demo-space-efficiency-1","volume_name_prefix":"demo-prefix-1"},
+                  "demo-system-id-2":{"pool":"demo-pool-2","volume_name_prefix":"demo-prefix-2"}}'  # Optional.
+  pool: demo-pool
+  SpaceEfficiency: demo-space-efficiency  # Optional.
+
+  csi.storage.k8s.io/secret-name: demo-secret-with-config

```

Comment:
IMO `demo-secret-from-config` is better, but since we want to refer to this secret in other places, `demo-config-secret` is even better

Reply:
done